1.3.0/publish.py

```diff
@@ -0,0 +1,19 @@
+import audb
+
+previous_version = "1.2.0"
+version = "1.3.0"
+build_dir = "./build"
+
+repository = audb.Repository(
+    name="data-public-local",
+    host="https://artifactory.audeering.com/artifactory",
+    backend="artifactory",
+)
```

Comment:
Please publish to the public repository:

```suggestion
repository = audb.Repository(
    name="data-public",
    host="https://audeering.jfrog.io/artifactory",
    backend="artifactory",
)
```

It's now also fine to continue with this pull request.